requirements.txt

```diff
@@ -8,3 +8,5 @@ Pygments~=2.1
 PyPrint~=0.2.6
 requests~=2.12
 setuptools>=17.0
+attrs~=16.3.0
+
```

Comment:
Travis was not recognizing attrs~=16.3.0 unless there was  a newline .It was saying module not found . Other CIs had no issues.